packages/server/src/course/ical.service.ts

```diff
@@ -1,30 +1,22 @@
 import { Injectable } from '@nestjs/common';
 import 'moment-timezone';
-import {
-  CalendarComponent,
-  CalendarResponse,
-  fromURL,
-  VEvent,
-} from 'node-ical';
 import { RRule } from 'rrule';
 import { Connection, DeepPartial } from 'typeorm';
 import { findOneIana } from 'windows-iana/dist';
-import { QueueModel } from '../queue/queue.entity';
 import { CourseModel } from './course.entity';
 import { OfficeHourModel } from './office-hour.entity';
 import moment = require('moment');
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import { Cron } from '@nestjs/schedule';
 import { RedisService } from 'nestjs-redis';
 import * as Redlock from 'redlock';
-import { SemesterModel } from '../semester/semester.entity';
 
 type Moment = moment.Moment;
 
-type CreateOfficeHour = DeepPartial<OfficeHourModel>[];
+type _CreateOfficeHour = DeepPartial<OfficeHourModel>[];
```

Comment:
out of pure curiosity, what do the underscore renaming do. i see a few times around the diffs

Reply:
nvm, ignore this. i just remembered that typescript is a thing

Reply:
Yeah we have a linting rule that won't let you make unused variables unless the name starts with a `_`